usecases/cluster/conflict_delegate.go

```diff
@@ -0,0 +1,83 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package cluster
+
+import (
+	"os"
+
+	"github.com/hashicorp/memberlist"
+	"github.com/hashicorp/raft"
+	"github.com/sirupsen/logrus"
+)
+
+type conflictDelegate struct {
+	logger  logrus.FieldLogger
+	localID string
+	raft    *raft.Raft
+	voter   bool
+}
+
+func (d *conflictDelegate) SetVoter(voter bool) {
```

Comment:
Let's make sure we have godoc on all the functions in that package for completeness :)